tarpc/src/serde_transport.rs

```diff
@@ -277,6 +277,168 @@ pub mod tcp {
     }
 }
 
+#[cfg(all(unix, feature = "unix"))]
+#[cfg_attr(docsrs, doc(cfg(all(unix, feature = "unix"))))]
+/// Unix Domain Socket support for generic transport using Tokio.
+pub mod unix {
+    use {
+        super::*,
+        futures::ready,
+        std::{marker::PhantomData, path::Path},
+        tokio::net::{unix::SocketAddr, UnixListener, UnixStream},
+        tokio_util::codec::length_delimited,
+    };
+
+    mod private {
+        use super::*;
+
+        pub trait Sealed {}
+
+        impl<Item, SinkItem, Codec> Sealed for Transport<UnixStream, Item, SinkItem, Codec> {}
+    }
```

Comment:
Haha funny you ask, because I asked myself the same. Implementing `Sealed` (a private trait) is usually so you can make that a bound on some other trait so no external crates can implement your (non-private) trait on their types. e.g. https://rust-lang.github.io/api-guidelines/future-proofing.html

However here it doesn't look like it's actually a bound anywhere?

I left it in because I didn't know if it was planned for the future, or there was some other reason.

Reply:
I think you can remove it for now; it's easy enough to add back if needed!

Reply:
Removed in: (2143f75)[https://github.com/google/tarpc/pull/380/commits/2143f758f1a6c5043b43de28556fb05f0b09d77d]